app/views/html_attachments/_print_meta_data.html.erb

```diff
@@ -0,0 +1,40 @@
+<br/>
+<br/>
+<br/>
+<p>
+<%= image_tag "open-government-licence.png", id: "meta-data-licence" %>
+</p>
+<br/>
+<br/>
+<br/>
+<p>
+  © Crown copyright <%= Date.today.year %>
+</p>
+<p>
+  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gsi.gov.uk" class="meta-data-email">psi@nationalarchives.gsi.gov.uk</a>.
+</p>
+<p>
+  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
+</p>
+<p>
+  This publication is available <a href="www.gov.uk/government/publications">www.gov.uk/government/publications</a>
+</p>
+<% unless @html_attachment.meta_data_contact_address.blank? %>
+  <p>
+    Any enquiries regarding this publication should be sent to us at:<br/>
+  </p>
+  <%= simple_format(@html_attachment.meta_data_contact_address) %>
+<% end %>
+
+<% unless @html_attachment.isbn.blank? %>
+  <p>
+    Web ISBN: <%= @html_attachment.isbn %>
+  </p>
+<% end %>
+<p>
+  Printed on paper containing 75% recycled fibre content minimum
+</p>
+<p>
+  Printed in the UK by the Williams Lea Group on behalf of the Controller of Her Majesty's Stationery Office
+</p>
```

Comment:
Worth checking with content team/whomever requested the story, but these last two paragraphs seem out of place given they're not going to be true for anyone that visits the website and prints the html attachment on their printer at home.